core/src/main/java/org/infinispan/container/entries/ReadCommittedEntry.java

```diff
@@ -116,6 +118,17 @@ public boolean isNull() {
 
    @Override
    public final void commit(DataContainer container) {
+      realCommit(-1, container);
```

Comment:
Instead of manually typing `-1`, should we be using `SegmentSpecificCommand.UNKOWN_SEGMENT`?

Reply:
This applies to other parts of the code as well

Reply:
Whoops thought I got all of them. Let me double check again.